guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>We could mention that input_features could be both feature layers and spatially enabled data frames just like we have for distance features.</p><p> </p><p>The cell sizes while training and predicting can vary.</p><p>While training - We drill down to the location in case of points, In case of polygons we take average for continuous and majority for categorical.</p><p>While predicting - We convert all rasters to the minimum cell size in the set and then pick up the values.</p><p>We handle different spatial references as well for training and inferencing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>